tests/components/unifiled/test_config_flow.py

```diff
@@ -0,0 +1,114 @@
+"""Test the unifiled config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.unifiled.config_flow import CannotConnect
+from homeassistant.components.unifiled.const import DOMAIN
+
+from tests.common import mock_coro
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == "form"
+    assert result["errors"] == {}
+
+    with patch(
+        "homeassistant.components.unifiled.config_flow.unifiled.getloginstate",
+        return_value=mock_coro(True),
```

Comment:
`getloginstate` is not a coroutine, so we should just set the `return_value` to `True`. Don't use `mock_coro`.